softvis3d-frontend/package.json

```diff
@@ -14,7 +14,7 @@
     "prebuild:ci": "yarn run lint -s && yarn run test:coverage -s && yarn run test:sonarqube",
     "build:ci": "yarn run build:force -s",
     "build:force": "webpack --progress --colors --prod",
-    "lint": "tslint \"src/**/*.ts?(x)\" && shx echo \"[tslint] Lint successful\"",
+    "lint": "tslint \"src/**/*.ts?(x)\" && tslint \"test/**/*.ts?(x)\" && shx echo \"[tslint] Lint successful\"",
```

Comment:
I'm ok with doing it seperately, but we should do it. Test code should have the standard as source code. 
Removed test linting from src lint script and added a pretest step for the test linting. Is this ok?